src/transformers/models/segformer/modeling_segformer.py

```diff
@@ -58,18 +57,8 @@
 ]
 
 
-# Inspired by
-# https://github.com/rwightman/pytorch-image-models/blob/b9bd960a032c75ca6b808ddeed76bee5f3ed4972/timm/models/layers/helpers.py
-# From PyTorch internals
-def to_2tuple(x):
-    if isinstance(x, collections.abc.Iterable):
-        return x
-    return (x, x)
```

Comment:
This is still used at line 106 and 107 on master, and your PR doesn't seem to change that.